### ISSUE TITLE
Can't scroll up in Gradio app deployed in HF space

### ISSUE THREAD

**Issue by SkalskiP** [gradio-app/gradio#6587]
### Describe the bug

After deploying Gradio app in HF space, you can't scroll up. When I run the Gradio app locally, everything works. The problem only appears in Chrome. When I run in Safari everything works.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

The problem is apparent in this Gradio app: https://huggingface.co/spaces/Roboflow/SoM

### Screenshot

https://github.com/gradio-app/gradio/assets/26109316/3ca88f5f-212a-4df7-9d2b-18ab86320407

### System Info

```shell
`3.50.2`, but I tied it with `4.x`, and the problem persists.
```

### Severity

Blocking usage of gradio

**Comment by SkalskiP** [comment]
Hi! 👋🏻 The issue still persists. Is there any chance you could help us with this bug? I'd really appreciate your assistance.

**Comment by freddyaboulton** [comment]
I think it's some issue with the Dockerfile? Doesn't happen for regular gradio-sdk spaces. @pngwn @hannahblair something is up with the iframe - do you have any idea what?

**Comment by SkalskiP** [comment]
Oh, that's interesting. I used the same Dockerfile with my other spaces. :/

**Comment by pngwn** [comment]
I don't know why this is happening but the root cause is that the iframe isn't being resized correctly. You can confirm this by going to the page and changing the iframe size manually in the dev tool (if you make it about 1500px it work as expected).

I thought i had fixed this once for all but i'll fix this once and for all again.

**Comment by freddyaboulton** [comment]
Thanks @pngwn !

**Comment by SkalskiP** [comment]
Thanks @pngwn! Should I do anything on my side? Or should I just wait for your fix?

**Comment by SkalskiP** [comment]
Awesome! Thank you! 🙏🏻

**Comment by kieranfraser** [comment]
Is there a fix for this? I'm having the same issue [here](https://huggingface.co/spaces/Trusted-AI/art-huggingface-evasion) (but do not use a Dockerfile)

**Comment by pngwn** [comment]
Yeah it is to do with the length of the contents, especially when they are difficult to measure. 

We have tried to fix this several times using different approaches but it keeps coming back. I'll take a look and see if I can come up with something else.

**Comment by abidlabs** [comment]
We're looking into this issue, but I'll close in favor of a duplicate: https://github.com/gradio-app/gradio/issues/10826
